docs/topics/multiplatform/multiplatform-dsl-reference.md

```diff
@@ -400,6 +400,10 @@ kotlin {
 
                 // A shortcut for includeDirs.allHeaders.
                 includeDirs("include/directory", "another/directory")
+
+                // Header files to be included in the bindings.
```

Comment:
The table above still mentions "cinterop tool arguments" when describing these properties. Is this intended?

Reply:
True, missed it, now it's consistent